examples/next/pages/ui/components/amplify-authenticator/sign-in-with-email/index.page.tsx

```diff
@@ -0,0 +1,15 @@
+import { AmplifyAuthenticator, translations } from '@aws-amplify/ui-react';
```

Comment:
This could become:

```suggestion
import { AmplifyAuthenticator } from '@aws-amplify/ui-react/legacy';
```

Reply:
what would be the actual benefit? Cleaner exports?

Reply:
Yes, but mostly (per the description) allowing us to re-export the new `withAuthenticator` **and** the legacy `withAuthenticator`.

This would give you 1 package with both new & legacy to use interchangeably.

Reply:
got it, thanks!